automation_tools/__init__.py

```diff
@@ -604,6 +604,63 @@ def setup_foreman_discovery():
     run('rm -rf {0}'.format(template_file))
 
 
+def setup_foreman_discovery_70(upstream=False):
```

Comment:
I think we should merge `setup_foreman_discovery` and `setup_foreman_discovery_70` into one method.
We can pass in 2 args , sat_version and upstream to control the flow, in that single method, this way may be can reduce the unwanted code.